node.js

```diff
@@ -84,6 +84,7 @@ module.exports = {
     'space-infix-ops': error,
     'prefer-const': warn,
     'prefer-rest-params': error,
-    'prefer-spread': error
+    'prefer-spread': error,
+    'no-unused-vars': [ error, { argsIgnorePattern: '^_' } ]
```

Comment:
it should use => https://eslint.org/docs/rules/no-unused-vars#args
`{ "args": "after-used" }`

Reply:
That would result in a different behaviour as far as I can tell.